node-src/lib/turbosnap/compareBaseline.test.ts

```diff
@@ -13,115 +13,61 @@ const getContext: any = (baselineCommits: string[]) => ({
   git: { baselineCommits },
 });
 
-describe('compareBaseline', () => {
-  it('finds changed dependency names', async () => {
-    const ctx = getContext();
-    const headDependencies = await getDependencies(ctx, {
-      rootPath: path.join(__dirname, '../../__mocks__/dependencyChanges/react-async-10'),
+async function getMockedDependencies(headName: string, baseName: string) {
+  const ctx = getContext();
+  return {
+    head: await getDependencies(ctx, {
+      rootPath: path.join(__dirname, `../../__mocks__/dependencyChanges/${headName}`),
       manifestPath: 'package.json',
       lockfilePath: 'yarn.lock',
-    });
-    const baselineChanges = await compareBaseline(ctx, headDependencies, {
-      rootPath: path.join(__dirname, '../../__mocks__/dependencyChanges/react-async-9'),
+    }),
+    base: await getDependencies(ctx, {
+      rootPath: path.join(__dirname, `../../__mocks__/dependencyChanges/${baseName}`),
       manifestPath: 'package.json',
       lockfilePath: 'yarn.lock',
-    });
+    }),
+  };
+}
+
+describe('compareBaseline', () => {
+  it('finds changed dependency names', async () => {
+    const { head, base } = await getMockedDependencies('react-async-10', 'react-async-9');
```

Comment:
What would you think about turning the argument to `getMockedDependencies` into an object so it's a bit easier to tell what's going on? I actually initially assumed it was base then head 😬. Small change but something like this:

```
getMockedDependencies({ head: 'react-async-10', base: 'react-async-9' });
```

Reply:
It's actually calls in order, so in tests further down it's head root, then head subdir, then bases.  It's a little hard to encode that.  Open to ideas though.

Reply:
Could just pop comments on all over though.

Reply:
Once I open the test file by itself, it actually makes more sense than the diff. I'm cool with leaving them as positional argument!

Reply:
It could become more verbose where it maps results to file paths I guess.  I updated the comments to start.